pages/wiki/ble-profiles.md

```diff
@@ -3,140 +3,183 @@ title: Bluetooth Low Energy Profiles
+</removed>
+```
+
+**Notification Feedback Characteristic (UUID: 00009002-0000-0000-0000-00A57E401D05)**
+
+Eventually, this characteristic should allow the server to send command regarding notifications to the client (e.g: close a notification or call an action) but this feature is not standardized nor documented yet. Don't hesitate to contribute!
+
+# Screenshot Profile
+
+A client can save the content of the server's screen using the Screenshot profile.
+
+### Screenshot Service (UUID: 00006071-0000-0000-0000-00A57E401D05)
+**Screenshot Request Characteristic (UUID: 00006001-0000-0000-0000-00A57E401D05)**
+
+This characteristic can be written by the client to any value (for example one byte set to 0) to trigger a byte stream on the Screenshot Content Characteristic.
+
+**Screenshot Content Characteristic (UUID: 00006002-0000-0000-0000-00A57E401D05)**
+
+This characteristic can be used to be notified of a screenshot content. After a write to the Screenshot Request Characteristic, the first value signaled by the Screenshot Content Characteristic should be made of four bytes containing the size of the data to be transmitted and can be used to show a progress bar during the screenshot download time. The rest of the transmission is made of chunks of raw data to be concatenated by the client in a buffer to reconstruct a screenshot of the watch's screen. The complete buffer should be a valid JPEG file but other formats could also be considered in the future.
+
+# Time Profile
+
+A client can change the server's system time using the Time Profile. This should be used periodically to synchronize a watch's time with the device it is currently connected to.
+
+### Time Service (UUID: 00005071-0000-0000-0000-00A57E401D05)
+**Time Set Characteristic (UUID: 00005001-0000-0000-0000-00A57E401D05)**
+This characteristic can be written. It is made of 6 bytes:
+
+ - *Year*: (byte 0) Year - 1900. 
+ - *Month*: (byte 1) A value in the range \[0-11\] where 0 = January and 11 = December
+ - *Day*: (byte 2) A value in the range \[1-31\]
+ - *Hour*: (byte 3) A value in the range \[0-23\]
+ - *Minute*: (byte 4) A value in the range \[0-59\]
+ - *Second*: (byte 5) A value in the range \[0-59\]
+
+# Weather Profile
+
+A client can fill weather information buffers on the watch using the Weather Profile. Those information can typically be used by asteroid-weather or asteroid-launcher's Today page to show current info.  For each of the characteristics except for the Weather City Characteristic, the data is interpreted as a sequence of five two-byte big-endian values that correspond to "today" through "today + 4 days".
+
+### Weather Service (UUID: 00008071-0000-0000-0000-00A57E401D05)
+
+**Weather City Characteristic (UUID: 00008001-0000-0000-0000-00A57E401D05)**
+
+This characteristic can be written by the client to set the name of the city whose weather information are attached to. Values should be UTF-8 strings and they do not need to be NUL-terminated.
```

Comment:
Typo here: This should be `NULL-terminated`

Reply:
It's confusing, but `NULL` and `NUL` are two different things.  `NULL` is a null pointer, while `NUL` is the name of a character with the value of zero.  ASCII strings are often terminated with a `NUL` character, and I wanted to point out the fact that such a terminator is **NOT** required here.  I'm open to suggestions for better wording.

Reply:
I did not know that, thanks for this!

Your example of ASCII strings doesn't seem to be consistent looking at my first search results (`NUL` and `NULL` appear to be mixed and matched). Regardless, it does show the difference very well.